topdown/eval.go

```diff
@@ -376,6 +376,27 @@ func (e *eval) evalStep(iter evalIterator) error {
 			}
 			return nil
 		})
+	case *ast.Every:
+		// TODO: Partial
+		// TODO: think about indexing comprehension...?
+		key := terms.Key
+		if key == nil {
+			key = e.generateVar(fmt.Sprintf("term_%d_%d", e.queryID, e.index)) // TODO: move into rewriting?
```

Comment:
+1 seems like it would make sense to move this into rewriting.

Reply:
✔️ #4326 I've made a quick process on that one.